docs/cli/server.md

```diff
@@ -146,6 +146,8 @@ The ```bor server``` command runs the Bor client.
 
 - ```v5disc```: Enables the experimental RLPx V5 (Topic Discovery) mechanism (default: false)
 
+- ```txarrivalwait```: Maximum number of milliseconds to wait before requesting an announced transaction (default: 100)
```

Comment:
let's keep it 500 by far.

Reply:
updated default to 500:

commit [b5ff754b7](https://github.com/thogard785/bor/commit/b5ff754b70c15fa0955173b3d6ea660bce3b1fa2)